docs/source/quickstart.rst

```diff
@@ -109,56 +26,36 @@ you may or may not have the examples already.
 Location
 --------
 
-1. If you installed Chaco as part of Canopy, the location of the examples depends
-   on your platform:
+The examples are located in the :file:`examples/` subdirectory inside the root
+of the Chaco source tree, next to :file:`docs/` and the :file:`chaco/`
+directories. You can download the source code available on 
+`Github <https://github.com/enthought/chaco>`_ or download a source
+distribution from PyPi. If you don't want a local copy of the Chaco codebase,
+you can browse the  
+`examples on GitHub <https://github.com/enthought/chaco/tree/master/examples>`_.
```

Comment:
This information is currently correct, but will need to be updated once #535 is addressed